### ISSUE TITLE
Option to hide monitors on projects

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6530]
### Why this would be helpful

Oftentimes, monitors can get in the way of things in projects that are not well thought through. A feature that hides adds a button that allows temporary hiding of all monitorswould be helpful.

### How this addon works

This addon adds a button to the top bar (e.g. where the green flag, stop sign, etc are) that toggles the visibility state of monitors.

**Comment by WorldLanguages** [comment]
Hmm, theoretically this could let you click below the monitors on sprites you aren't supposed to click.

**Comment by mybearworld** [comment]
> Hmm, theoretically this could let you click below the monitors on sprites you aren't supposed to click.

You could set `opacity: 0` and `pointer-events: none` to avoid this. This would probably be confusing UX, though...

**Comment by DNin01** [comment]
There are also times where you _do_ want to see the monitors even though they get in the way. What then? And WorldLanguages just said above that you might be able to click through them depending on how we go about it. We probably don't want to change project behavior in ways like this.

#3912 partially resolves the getting in the way part by allowing you to set the colors to transparent values. I think that might be a better solution.

**Comment by DNin01** [comment]
> > Hmm, theoretically this could let you click below the monitors on sprites you aren't supposed to click.
> 
> 
> 
> You could set `opacity: 0` and `pointer-events: none` to avoid this. This would probably be confusing UX, though...

Yeah, the monitors could eat up your clicks while they were hidden.

**Comment by mybearworld** [comment]
> There are also times where you _do_ want to see the monitors even though they get in the way. What then?

Then you could press the button again.

> We probably don't want to change project behavior in ways like this.

With the approach above we wouldn't be changing behavior.

**Comment by TheColaber** [comment]
why is this useful?
